webpack.config.js

```diff
@@ -179,7 +179,15 @@ if (TARGET_ENV === 'development') {
         devServer: {
             inline: true,
             stats: 'errors-only',
-            contentBase: path.join(__dirname, "src/static/assets")
+            contentBase: path.join(__dirname, "src/static/assets"),
+            proxy: {
+                '/server': {
```

Comment:
And what happends in the devServer is that all calls to /server are intercepted, and rerouted like configured below the comment

Reply:
Clever!